supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -520,3 +520,14 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 *Incorrect forms*:
 
 *See also*:
+
+[discrete]
+[[bytecode]]
+==== image:images/yes.png[yes] bytecode (noun)
+*Description*: A _bytecode_ is a non-human-readable instruction set generated by a compiler. Bytecode is typically either executed by a virtual machine (VM) or compiled again into machine code.
```

Comment:
xompiled again > re-compiled ?

Reply:
@dfitzmau Although 'recompile' is a term, it's not frequently used and is what I consider to be technical and archaic. While this glossary entry is a technical definition, I'd prefer not to use 'recompile' if there's a way to write around it.

Reply:
Hi @mportman12 . NP. I guess it depends on what product one works on. In the OpenJDK "re-compile" is a common action when building Java applications·

Reply:
The term "recompile" sounds more natural to me, but maybe I'm archaic too :rofl: 
I agree that it's a technical term, but it's used in a description of a technical term so it's appropriate?

Note that if we do use this term, the spelling should be "recompile" (without the hyphen), as specified in the Hyphens topic and in Merriam-Webster.

Reply:
Ok, if 'recompile' is not archaic in this context, I'll update the text for it.